src/overlays/actors/ovl_Bg_Umajump/z_bg_umajump.h

```diff
@@ -3,11 +3,21 @@
 
 #include "global.h"
 
+#define BG_UMAJUMP_GET_OBJECT_INDEX(thisx) (((thisx)->params >> 8) & 0xFF)
+
```

Comment:
There are a lot of times where params gets compared to values. Even if they aren't documented yet, it would still be beneficial to create the enum now and use it.

Reply:
Where is the value? are you referring to the 255 part or the 8 ?

Reply:
So I'm realizing this was actually a terrible place to put the comment, my bad. However in the `.c` file there are a bunch of `this->dyna.actor.params == x` where `x` is just some number. These `x`s should get an enum. (I'll make a comment pointing one or more out).